dockers/docker-database/docker-database-init.sh

```diff
@@ -18,6 +18,7 @@ then
 fi
 
 REDIS_DIR=/var/run/redis$NAMESPACE_ID
+chown -R redis:redis $REDIS_DIR
```

Comment:
>chown [](http://example.com/codeflow?start=0&length=5)

`chown` line is too early, files created later will be owned by root:root.

Reply:
fixed, moved to bottom of file